### PR TITLE
Encode password for database URI

### PR DESCRIPTION
**Pull request by rjweeks**
# Motivation and Context
Special characters in the database password prevent SQLAlchemy working with a database URI string.

# What has changed
URL Encode database password

# How to test?
Test following instructions in GitHub ReadMe

# Links

https://trello.com/c/ps4GpVHf/914-bug-qid-batch-runner-cant-handle-special-characters-in-db-password

https://docs.sqlalchemy.org/en/13/core/engines.html

### CODE REVIEW

**Review by jcox-dev — APPROVED** [review]

**Review by jcox-dev — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -61,6 +61,26 @@ pipenv run python generate_qid_batch.py <path to config csv>
...
+export SFTP_PASSPHRASE=secret
+export SFTP_DIRECTORY="Documents/sftp/"
+export SFTP_KEY_FILENAME="dummy_keys/dummy_rsa"
+export OUR_PUBLIC_KEY_PATH="dummy_keys/our_dummy_public.asc"
+export OTHER_PUBLIC_KEY_PATH="dummy_keys/supplier_dummy_public.asc"
+EOF
+```
+
+```bash
+Create local output directory directory
```
Comment (jcox-dev) [inline README.md]:
```suggestion
Create local output directory, e.g.:
```

File: README.md [hunk 2]
```diff
@@ -61,6 +61,26 @@ pipenv run python generate_qid_batch.py <path to config csv>
...
+e.g mkdir print_files
```
Comment (jcox-dev) [inline README.md]:
```suggestion
mkdir print_files
```

**Review by jcox-dev — CHANGES_REQUESTED** [review]

**Review by ryangrundy7 — APPROVED** [review]
LGTM 👍

File: README.md (hunk 1 above)
Comment (rjweeks) [reply README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -61,6 +61,25 @@ pipenv run python generate_qid_batch.py <path to config csv>
...
+export SFTP_PASSPHRASE=secret
+export SFTP_DIRECTORY="Documents/sftp/"
+export SFTP_KEY_FILENAME="dummy_keys/dummy_rsa"
+export OUR_PUBLIC_KEY_PATH="dummy_keys/our_dummy_public.asc"
+export OTHER_PUBLIC_KEY_PATH="dummy_keys/supplier_dummy_public.asc"
+EOF
+```
+Create local output directory
+```bash
+mkdir print_files++++++++
```
Comment (jcox-dev) [inline README.md]:
```suggestion
mkdir print_files
```

**Review by jcox-dev — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (rjweeks) [reply README.md]: Deleted

**Review by jcox-dev — APPROVED** [review]

**Pull request merged**